web_src/js/features/repo-issue-list.js

```diff
@@ -198,8 +198,9 @@ async function initIssuePinSort() {
 }
 
 export function initRepoIssueList() {
-  if (!document.querySelectorAll('.page-content.repository.issue-list, .page-content.repository.milestone-issue-list').length) return;
+  if (!document.querySelectorAll('.page-content.repository.issue-list, .page-content.repository.milestone-issue-list, .page-content.dashboard.issues').length) return;
   initRepoIssueListCheckboxes();
-  initRepoIssueListAuthorDropdown();
+  initRepoIssueListUserDropdown('#authorDropdown');
+  initRepoIssueListUserDropdown('#assigneeDropdown');
```

Comment:
Would call this `initRepoIssueListUserDropdowns` and then iterate the results of the `.user-remote-search`. No need to really specify these ids in js (and can probably remove the IDs from JS).

Reply:
Thanks, looks good.